crates/subspace-core-primitives/src/lib.rs

```diff
@@ -215,6 +222,63 @@ impl PosProof {
     pub const SIZE: usize = 17 * 8;
 }
 
+/// Proof of time key(input to the encryption).
+#[derive(Debug, Copy, Clone, From, Into, Encode, Decode)]
+pub struct PotKey(PotBytes);
+
+/// Proof of time seed (input to the encryption).
+#[derive(Debug, Copy, Clone, From, Into, Encode, Decode)]
+pub struct PotSeed(PotBytes);
+
+/// Proof of time ciphertext (output from the encryption).
+#[derive(Debug, Copy, Clone, From, Into, Encode, Decode)]
+pub struct PotCheckpoint(PotBytes);
+
+/// Proof of time.
+/// TODO: versioning.
+#[derive(Debug, Clone, Encode, Decode)]
+pub struct PotProof {
+    /// Slot the proof was evaluated for.
+    pub slot_number: SlotNumber,
+
+    /// The seed used for evaluation.
+    pub seed: PotSeed,
```

Comment:
I have reviewed the spec and `key` is also necessary to include in the PotProof otherwise it's unverifiable. Spec is [updated](https://www.notion.so/subspacelabs/PoT-Specification-WIP-4e17a6d5b03a4abea864ba2d0b97970e?pvs=4#0dd384cfc85942bcb2c644710516e1ad).

Reply:
Done